scripts/kfctl.sh

```diff
@@ -142,6 +142,7 @@ ksApply() {
   ks apply default -c spartakus
   ks apply default -c argo
   ks apply default -c pipeline
+  ks apply default -c modeldb
```

Comment:
Can we hold off installing modeldb by default? I think it would be good to get some usage as an optional package and then use that to install it by default if it makes sense.

Reply:
Done